packages/billing/lib/billing.ts

```diff
@@ -1,10 +1,10 @@
+import tracer from 'dd-trace';
 import { uuidv7 } from 'uuidv7';
 
 import { Err, Ok, flagHasUsage, report } from '@nangohq/utils';
 
 import { Batcher } from './batcher.js';
 import { envs } from './envs.js';
-import { logger } from './logger.js';
```

Comment:
Billing being a `library` (and not an `app`). I prefer to delegate the the logging to whoever is using the library.